webapp/backend/lightchanger/views.py

```diff
@@ -75,9 +75,9 @@ def partial_update(self, request, *args, **kwargs):
     @basic_authentication
     def reset_parameters(self, request, *args, **kwargs):
       light_pattern_json = request.data
-      light_pattern_name = light_pattern_json['light_pattern_name']
       light_pattern_id = light_pattern_json['light_pattern_id']
       existing = LightPatternOption.objects.get(pk=light_pattern_id)
+      light_pattern_name = existing.animation_id
```

Comment:
@tstentz Repeating what I commented on the last pr here: Made a change to the backend api that removes the redundant `light_pattern_name` arg. This greatly simplifies our reducers since they now won't have to go into the current state to retrieve the animation name.